autoload/kite.vim

```diff
@@ -87,24 +87,28 @@ endfunction
 
 
 function! kite#bufenter()
-  if s:supported_language()
+  if kite#languages#supported_by_plugin()
     call s:launch_kited()
 
-    call s:disable_completion_plugins()
-    call s:setup_options()
-    call s:setup_events()
-    call s:setup_mappings()
+    if kite#languages#supported_by_kited()
```

Comment:
Is my understanding of the new logic correct?
- the plugin has a flag for the enabled languages, this defaults to Python=on and Go=off
- Support is only enabled if both the plugin and kited report a language as supported
- kited's language support is only requested once for the current process

Reply:
Yes, that's all correct.